server/taskflows/hpccloud/taskflow/paraview/visualizer.py

```diff
@@ -182,7 +182,7 @@ def upload_input(task, cluster, job, *args, **kwargs):
     if file_id:
         file_id = file_id[0].value
         task.logger.info('Visualizing file ID: %s' % file_id)
-        job['params']['dataDir'] = '.'
+        job['params']['dataDir'] = '$PWD'
 
```

Comment:
$PWD remain a string till the execution on the cluster. Moreover, the --data expect a full path, so '.' never worked.